pkg/cmd/server/server.go

```diff
@@ -41,6 +42,7 @@ var (
 	schemaFile         = flag.String("schema-file", "", "schema-file")
 	loadServerDataFlag = flag.Bool("load-server-data", false, "load-server-data")
 	pidfile            = flag.String("pid-file", "", "Name of file that will hold the pid")
+	cpuprofile         = flag.String("cpuprofile", "", "write cpu profile to file")
```

Comment:
Can you rename the flag cpuprofile -> cpu-profile
and the variable cpuprofile -> cpuProfile (the same with pidfile --> pidFile)